README.md

```diff
@@ -1,69 +1,125 @@
+| [theiaide/theia-full](https://hub.docker.com/r/theiaide/theia-full) | Theia-based example application with all available published applications | |
+| [theiaide/theia-go](https://hub.docker.com/r/theiaide/theia-go) | Theia-based Go example application | [docs](./theia-go-docker/README.md) |
+| [theiaide/theia-java](https://hub.docker.com/r/theiaide/theia-java) | Theia-based Java example application | |
+| [theiaide/theia-python](https://hub.docker.com/r/theiaide/theia-python) | Theia-based Python example application | [docs](./theia-python-docker/README.md) |
+| [theiaide/theia-ruby](https://hub.docker.com/r/theiaide/theia-ruby) | Theia-based Ruby example application | |
+| [theiaide/theia-rust](https://hub.docker.com/r/theiaide/theia-rust) | Theia-based Rust example application | [docs](./theia-rust-docker/README.md) |
+| [theiaide/theia-swift](https://hub.docker.com/r/theiaide/theia-swift) | Theia-based Swift example application | |
+| [theiaide/theia](https://hub.docker.com/r/theiaide/theia) | Theia-based JavaScript/TypeScript (Web Technologies) example application | |
+| [theiaide/yangster](https://hub.docker.com/r/theiaide/yangster) | Theia-based JavaScript/TypeScript (Web Technologies) example application | |
+
+---
+
+<br />
+
+### Additional Docker Examples
+
+| Image Name | Description | Documentation |
+|:---|:---|:---|
+| `theia-deb-build-docker` | Example on how to package the IDE into a Debian package | [docs](./theia-deb-build-docker/README.md) |
+| `theia-https-docker` | Example on how to add security layer over existing images | [docs](./theia-https-docker/README.md) |
+| `theia-openshift-docker` | Example image for OpenShift | |
+| `theia-rpm-build-docker` | Example on how to package the IDE into an RPM | [docs](./theia-rpm-build-docker/README.md) |
 
-[![dockeri.co](http://dockeri.co/image/theiaide/theia)](https://hub.docker.com/r/theiaide/theia/)
+<br />
 
-`theiaide/theia` image contains an example of Theia based IDE for Web Developers.
+### Electron Apps
 
-### How to use `theiaide/theia` image?
+<img
+    src='./assets/electron.png'
+    alt='electron logo'
+    width="50px"
+/>
 
-At the moment Theia is still in [active development](https://github.com/theia-ide/theia#roadmap). It is recommended to use [`theiaide/theia:next`](#typefoxtheianext) image to have a look at the current state.
+| Application Name | Description | Documentation |
+|:---|:---|:---|
+| `theia-cpp-electron` | Theia-based C/C++ desktop IDE | [docs](./theia-cpp-electron/README.md) |
+| `theia-electron` | Theia-based JavaScript/TypeScript (Web Technologies) desktop IDE | [docs](./theia-electron/README.md) |
 
-This script pulls the image and runs Theia IDE on http://localhost:3000 with the current directory as a workspace. The option of `--init` is added to fix the [defunct process problem](https://github.com/theia-ide/theia-apps/issues/195).
+---
 
-    # Linux or macOS
```

Comment:
It is more important that a user know how to start and debug? Where does this information go? 

Information about building from sources should go to CONTRIBUTING.md.

Reply:
Do you propose we remove it, and instead link to `CONTRIBUTING.md`?
I believe an example already exists there on how to build from sources, debug, run.

Reply:
I think we should keep old content to how to start `theiaide` for running and debugging. And Move development content to `CONTRIBUTING.md`. referencing from README.md is fine if someone want to contribute or build own image